data/README.md

```diff
@@ -0,0 +1,40 @@
+# Datasets (This section is work in progress.)
+
+## Anomalies datasets
+
+There are several types of anomalies in data streams. We categorize them in the following way:
+- neatData – clean data sets generated with no anomalies included. Eg. sine, constant, spike train.
```

Comment:
- can use some "GH styling" like, `neatData/`
- remove the word "included" there